src/components/BundleTree/types.ts

```diff
@@ -1,12 +1,20 @@
 import { NestedToken } from '@unique-nft/substrate-client/tokens';
 
-export type INestingToken = NestedToken & {
+export interface INestingToken extends NestedToken {
   opened?: boolean
   disabled?: boolean
   selected?: boolean
 }
 
+export interface INestedTokenId {
```

Comment:
please use the one from SDK, we already have a billion instances of for this exact type